keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/annealing/ResourceActuator.kt

```diff
@@ -42,6 +42,8 @@ class ResourceActuator(
     val plugin = handlers.supporting(apiVersion, kind)
     val type = plugin.supportedKind.second
     val resource = resourceRepository.get(name, type)
+
+    val desired = plugin.desired(resource)
```

Comment:
We can now generate a desired state based on a spec. In the case of an image this means figuring out the specific app and base AMI versions